src/main/kotlin/no/nav/helse/sporenstreks/service/PostgresRefusjonskravService.kt

```diff
@@ -93,12 +92,15 @@ class PostgresRefusjonskravService(
                     val savedKvittering = kvitteringRepository.insert(
                             Kvittering(virksomhetsnummer = it.key,
                                     refusjonsListe = it.value,
-                                    tidspunkt = LocalDateTime.now())
-                            , con)
+                                    status = KvitteringStatus.JOBB,
+                                    tidspunkt = LocalDateTime.now()), con)
                     it.value.forEach { krav ->
                         krav.kvitteringId = savedKvittering.id
+                        krav.status = RefusjonskravStatus.JOBB
+                        lagreRefusjonskravJobb(krav, con)
                     }
-                    resultList.addAll(refusjonskravRepository.bulkInsert(it.value))
+                    lagreKvitteringJobb(savedKvittering, con)
+                    resultList.addAll(refusjonskravRepository.bulkInsert(it.value, con))
```

Comment:
denne er verdt å legge merke til, det er to forskjellige bulkinsert metoder, så vi må håpe begge to fungerer bra..